src/pages/VenuePage/VenuePage.tsx

```diff
@@ -204,12 +318,25 @@ export const VenuePage: React.FC = () => {
     venue.hasPaidEvents &&
     !isUserVenueOwner
   ) {
-    if (eventRequestStatus && !event) {
-      return <>This event does not exist</>;
+    if (isEventLoaded && !event) {
+      return (
+        <>
+          <Preload venue={venue} />
+          This event does not exist
+        </>
+      );
     }
 
     if (!event || !venue || !userPurchaseHistoryRequestStatus) {
-      return <LoadingPage />;
+      // considering there is prior !venue check, venue should exist at this point
+      return (
+        <>
+          <Preload venue={venue} />
+          <Suspense fallback={<></>}>
+            <LoadingPage />
+          </Suspense>
+        </>
+      );
```

Comment:
At this stage i'm not convinced that `Preload` is going to have any meaningful impact on things; and if we do decide to keep it, it shouldn't end up duplicated in so many places.

Reply:
Lazy loading `LoadingPage` and wrapping it with `Suspense` here doesn't make any sense.

As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
`Suspense` removed in 44282c08246eebc36473b0ce7cc4ae0e0f9482f7

Reply:
Preloading still in place with a hook, due to https://github.com/sparkletown/sparkle/pull/1713#discussion_r666071927